core/unit_test/TestViewSubview.hpp

```diff
@@ -1995,6 +1998,8 @@ void test_1d_assign() {
 
 template <class Space, class MemTraits = void>
 void test_2d_subview_3d() {
+  // FIXME_SYCL requires MDRange policy
```

Comment:
this comments out the entire test. Why not do it where the test is actually called, or even comment out the gtest alltogether so it doesn't look like this test is running. Same for the ones below. I mean these tests have basically 1-to-1 correspondents to actual gtest test definitions.

Reply:
Fixed. It was just more convenient to go through the one file than to figure out which test instantiates the non-compiling templates and see if the test can be disabled altogether. I see your point, though.